sourmash_lib/kmer_min_hash.hh

```diff
@@ -9,9 +9,17 @@
 #include <string>
 #include <unordered_map>
 
-// #include "kmer_hash.hh"
+#include "../third-party/smhasher/MurmurHash3.h"
 
-////
+uint64_t _hash_murmur(const std::string& kmer,
```

Comment:
This was defined in `sourmash_lib/_minhash.cc` before, so it wasn't visible when using a pure C++ program (only worked when header was compiled together with the extension...)